src/utility/Constants.js

```diff
@@ -1,4 +1,10 @@
 export const DB_COLLECTION = 'Hackathons'
 export const DB_HACKATHON = 'LHD2021'
 export const FAQ_COLLECTION = 'FAQ'
-export const DAYOF_COLLECTION = 'DayOf'
\ No newline at end of file
+export const NOTIFICATION_SETTINGS_CACHE_KEY = 'livesiteNotificationSettings'
+export const NOTIFICATION_PERMISSIONS = Object.freeze({
+  GRANTED: 'granted',
+  DEFAULT: 'default',
+  DENIED: 'denied',
+})
```

Comment:
was debating whether to put these variables here or in new util module `notifications.js`, thoughts?

Reply:
think it should be ok here for now, but if ever need to add more constants then it might be a good idea